frontend/src/data/loading/DataFetcher.ts

```diff
@@ -96,32 +96,48 @@ export class ApiDataFetcher implements DataFetcher {
       datasetId.startsWith("acs_2010_population")
     ) {
       result = result.map((row: any) => {
-        return { ...row, population: Number(row["population"]) };
+        return {
+          ...row,
+          population:
+            row["population"] == null ? null : Number(row["population"]),
+        };
       });
     } else if (datasetId.startsWith("cdc_restricted")) {
       result = result.map((row: any) => {
         return {
           ...row,
-          cases: Number(row["cases"]),
-          hosp_y: Number(row["hosp_y"]),
-          hosp_n: Number(row["hosp_n"]),
-          hosp_unknown: Number(row["hosp_unknown"]),
-          death_y: Number(row["death_y"]),
-          death_n: Number(row["death_n"]),
-          death_unknown: Number(row["death_unknown"]),
+          cases: row["cases"] == null ? null : Number(row["cases"]),
+          hosp_y: row["hosp_y"] == null ? null : Number(row["hosp_y"]),
+          hosp_n: row["hosp_n"] == null ? null : Number(row["hosp_n"]),
+          hosp_unknown:
+            row["hosp_unknown"] == null ? null : Number(row["hosp_unknown"]),
+          death_y: row["death_y"] == null ? null : Number(row["death_y"]),
+          death_n: row["death_n"] == null ? null : Number(row["death_n"]),
+          death_unknown:
+            row["death_unknown"] == null ? null : Number(row["death_unknown"]),
           population:
-            row["population"] == null ? null : Number(row["population"]),
+            row["population"] == null
+              ? null
+              : row["population"] == null
+              ? null
+              : Number(row["population"]),
```

Comment:
I think that this nested ternary is checking the same value twice `row["population"] == null`

Reply:
good catch! i think we can eliminate this whole weird null checking stuff because we are sending it in the correct type from BQ now (instead of strings to ships numbers)